extensions/1bookmark/tsconfig.json

```diff
@@ -0,0 +1,16 @@
+{
+  "$schema": "https://json.schemastore.org/tsconfig",
+  "include": ["src/**/*", "raycast-env.d.ts"],
+  "compilerOptions": {
+    "lib": ["ES2023"],
+    "module": "commonjs",
+    "target": "ES2022",
+    "strict": true,
+    "isolatedModules": true,
+    "esModuleInterop": true,
+    "skipLibCheck": true,
+    "forceConsistentCasingInFileNames": true,
+    "jsx": "react-jsx",
+    "resolveJsonModule": true
```

Comment:
@pernielsentikaer I Checked that you changed tsconfig.json and ran Check CI. This extension need to import server-side Types to build, so I need the following settings. Can you add these settings?

```json
    /* Path Aliases */
    "baseUrl": ".",
    "paths": {
      "@/*": ["./src/*"],
      "@repo/trpc-router": ["./types/trpc-router/index"],
    }
```

Reply:
You can try to add them again

Reply:
I kept the other parts as default settings and added only this part.